src/components/QuickSearchForm/__snapshots__/quick-search-form.spec.ts.snap

```diff
@@ -7,7 +7,7 @@ exports[`QuickSearchForm component > Renders a QuickSearchForm 1`] = `
   </p>
 </div>
 <div class="pdap-quick-search-form h-full max-h-[75-vh] justify-start p-0">
-  <form class="pdap-form flex flex-wrap gap-x-4" id="quick-search-form" name="quickSearchForm">
+  <form class="pdap-form flex flex-wrap gap-x-4 flex flex-wrap gap-x-4" id="quick-search-form" name="quickSearchForm">
```

Comment:
this isn't hurting anything, but maybe redundant? I would believe these are generated and applied to the form for some valid reason that would affect the snapshot, though.

Reply:
@josh-chamberlain No, that's important, actually, thank you for catching. It relates to `v-bind` [merge behavior](https://v3-migration.vuejs.org/breaking-changes/v-bind.html), and could cause problems in `Form` down the line. Making this update and one other.